apps/batchloader/batchLoader.js

```diff
@@ -293,7 +295,7 @@ function updateFileName(oldfileName) {
     let newFileName = $('#newFileName');
     let newName = newFileName.val();
     let fileExtension = newName.toLowerCase().split('.').reverse()[0];
-
+    alert(fileExtension);
```

Comment:
Did you indend this just for dev purposes?

Reply:
Yes, I used it(line 298) just for testing purposes..!

Reply:
We can just comment it out if we don't want to see an alter popup in the browser while uploading.

Reply:
```suggestion
```

Reply:
The code is modified as per the suggestions made .. with the commit message "Suggested Changes Made" ..!